File: java/src/raft/Par.java
package raft;

// Direccion host:puerto de un nodo del cluster. Simple, sin librerias.
public class Par {
    public final int id;
    public final String host;
    public final int puerto;

    public Par(int id, String host, int puerto) {
        this.id = id;
        this.host = host;
        this.puerto = puerto;
    }

    public String direccion() {
        return host + ":" + puerto;
    }
}

File: go/internal/raft/par.go
package raft

import "strconv"

// Par es la direccion host:puerto de otro nodo del cluster. Simple, sin
// librerias externas (equivalente a Par.java).
type Par struct {
	ID     int
	Host   string
	Puerto int
}

func (p Par) Direccion() string {
	return p.Host + ":" + strconv.Itoa(p.Puerto)
}
